modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js

```diff
@@ -232,20 +217,25 @@ export default function UpperToolbar({
 			xmlDefinition,
 		} = validXMLDefinition;
 
-		const publishedDefinitionResponse = await publishDefinitionRequest({
-			active,
-			content: xmlDefinition,
-			name,
-			title: definitionTitle,
-			title_i18n: definitionTitleTranslations,
-			version,
-		});
+		const publishedOrSavedDefinitionResponse =
+			await saveOrPublishDefinitionRequest({
+				active,
```

Comment:
I see `publishedOrSavedDefinition` and `saveOrPublishDefinition`. 

I believe we should standardize the order of terms.